README.md

```diff
@@ -1,14 +1,13 @@
-# copy-component
-Creates a copy element from existing element using data-copy attribute. Places a copy icon next to the element. When clicking the icon, the value of the element is copied and a copied overlay is displayed over the element for a short duration.\
-![Without Component Image](/screenshots/Component_1.png?raw=true "Without Component")\
-![With Component Image](/screenshots/Component_2.png?raw=true "With Component")\
-![After Copy Image](/screenshots/Component_3.png?raw=true "After Copy")
+# Blockonomics Copy Component
+Creates a copy element from existing element using data-copy attribute. Places a copy icon next to the element. When clicking the icon, the value of the element is copied and a copied overlay is displayed over the element for a short duration.
 
 ## Usage instructions
-* Copy the component JS, CSS, and Icons to your project
+* Copy the component JS, CSS, and Icons to your project or use a CDN
 * Load the component CSS in the header of your html
-`<link rel="stylesheet" href="css/components.css">`
+`<link rel="stylesheet" href="css/bnomics-input.css">`
```

Comment:
We have changed file names unnecessarily in this pull request. The copy component is a separate feature to Blockonomics, and bnomics-input is not a relevant name. Please revert file names for now, we can rename these in a separate PR if this is required.

Reply:
Will update in next Commit.

Reply:
it was renamed to ensure a consistent namespace when being server by CDN, the name's temporary and can be changed to whatever we decide on CDN

Reply:
This change does not relate to adding support for either <span> or <textarea> and should rather be changed in a separate PR with a focus on refactoring or this task alone.